### DISCUSSION TITLE
Obsidian style: callouts

### DISCUSSION THREAD

**Discussion by Praczet** [iamcco/markdown-preview.nvim#682] [Ideas]
Hi,

Firstly thank you for this plugin I really like it.

I would like to ask if it will be possible to add support for the Obsidian's callouts (markdown-it-obsidian-callouts)

Best regards

**Comment by pidgeon777** [comment]
I'm also very interested.

**Reply by Tweekism** [reply]
Tuurep is implementing GitHub callouts on Vivify right now, which seems like a subset of the obsidian ones...

**Reply by freqyfreqy** [reply]
I would also like to upvote this request as well!
(GitHub flavoured callouts) https://github.com/orgs/community/discussions/16925
